src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1885,8 +1902,7 @@ export class StudyViewPageStore {
     }
 
     @autobind
-    @action addCustomChart(newChart:NewChart) {
-        const uniqueKey = this.newCustomChartUniqueKey();
+    @action addCustomChart(newChart:NewChart, uniqueKey:string = this.newCustomChartUniqueKey(), newlyAdded:boolean=true) {
```

Comment:
After adding the parameter `newlyAdded`, this function will need to be called `updateCustomChart`

Reply:
instead it's better to rename `newlyAdded` to something else, may be `loadedByUserClick`?